proposals/0301-package-editing-commands.md

```diff
@@ -0,0 +1,168 @@
+# Package Editor Commands
+
+* Proposal: [SE-0301](0301-package-editing-commands.md) 
+* Authors: [Owen Voorhees](https://github.com/owenv)
+* Review Manager: [Tom Doron](https://github.com/tomerd)
+* Status: **Scheduled for Review (Feburary 4...February 15, 2021)**
```

Comment:
@owenv okay to schedule the review?

Reply:
@tomerd Sounds good to me, thanks! I need to make a few small edits now that SE-0294 has been accepted, but I can push those changes tonight.